lib/stream.js

```diff
@@ -36,7 +36,7 @@ export const makeAllStream = (spawned, {all}) => {
 
 // On failure, `result.stdout|stderr|all` should contain the currently buffered stream
 const getBufferedData = async (stream, streamPromise) => {
-	if (!stream) {
+	if (!stream || streamPromise === undefined) {
```

Comment:
A short code comment here would be great.

Reply:
Done :+1: